builtin/providers/vault/provider.go

```diff
@@ -90,6 +90,7 @@ func Provider() terraform.ResourceProvider {
 			"vault_auth_backend":   authBackendResource(),
 			"vault_generic_secret": genericSecretResource(),
 			"vault_policy":         policyResource(),
+			"vault_mount":          mountResource(),
```

Comment:
There is an existing `vault_auth_backend`; maybe `vault_secret_backend` is more consistent with the lingo.